### ISSUE TITLE
Brightness Hotkeys

### ISSUE THREAD

**Issue by Striker770** [jtaczanowski/ThinkPad-X201-Arrandale-macOS-OpenCore#22]
Hello, 

Any info on the Brightness Hotkeys? Having T410 here.

Thanks

**Comment by jtaczanowski** [comment]
hello @Striker770  on my x201 brightness hotkeys are working :)

**Comment by Striker770** [comment]
> hello @Striker770 on my x201 brightness hotkeys are working :)

Does the Brightness slider appear and is functional?

Can you provide the steps on the Brightness?

**Comment by jtaczanowski** [comment]
No, brightness slider is not appearing and also I can't see it in screen settings in system preferences - but hotkeys are changing brightness.

**Comment by rivciks** [comment]
I have also a t410 model here and can confirm that brightness hotkeys are not working :(
Is there a solution?
